package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "setup-matlab-action",
     "author": "The MathWorks, Inc.",
-    "version": "1.0.1",
+    "version": "1.2.3",
```

Comment:
This gets set to 2 on release?

Reply:
Yeah, usually we don't have to change this manually because the CI pipeline does it on release. It might be a bit weird this time because we're not using the usual `X.Y.Z` type version.